client/src/components/i18n/message.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import jQuery from 'jquery';
+
+// @see https://github.com/yahoo/react-intl/blob/v2.4.0/src/components/message.js
+const Message = ( { id, placeholders = [] } ) => {
```

Comment:
nitpick, Is there a better name for this? `Message` doesn't really reflect that this will print out translations

Reply:
That's alright. I can't think of something better tbh